infra/docker/Dockerfile

```diff
@@ -19,7 +18,9 @@ LABEL description="Madara, a blazing fast Starknet sequencer" \
   documentation="https://docs.madara.wtf/"
 
 COPY --from=builder /madara/target/release/madara /
+COPY --from=builder /madara/infra/chain-specs /chain-specs
 # 9444 JSON-RPC server
 # 9615 Prometheus exporter
+# 30333 P2P communication
 EXPOSE 9933 9944 9615 30333
```

Comment:
@abdelhamidbakhta mentioned rpc and ws are both on 9933 now. so we can close 9944 and update the comment on line 22